patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1,6 +1,17 @@
 --- ../src-base/minecraft/net/minecraft/block/Block.java
 +++ ../src-work/minecraft/net/minecraft/block/Block.java
-@@ -52,11 +52,12 @@
+@@ -1,10 +1,7 @@
+ package net.minecraft.block;
+ 
+-import com.google.common.collect.Sets;
```

Comment:
Patches must not change any imports, please make sure to leave these as they were.